addons/stock/static/tests/counted_quantity_widget.test.js

```diff
@@ -0,0 +1,69 @@
+import { defineMailModels } from "@mail/../tests/mail_test_helpers";
+import { expect, test } from "@odoo/hoot";
+import { contains, defineModels, fields, models, mountView } from "@web/../tests/web_test_helpers";
+
+class Quant extends models.Model {
+    quantity = fields.Float();
+    inventory_quantity = fields.Float({ string: "Counted quantity" });
```

Comment:
remove `_onChanges` statement below
```suggestion
    inventory_quantity = fields.Float({
        string: "Counted quantity",
        onChange: (quant) => {
            quant.inventory_diff_quantity = quant.inventory_quantity - quant.quantity;
        },
    });
```